maquetado/scss/footer.scss

```diff
@@ -0,0 +1,33 @@
+.footer {
+  align-items: center;
+  background-color: #00ADEE;
+  display: flex;
+  flex-direction: column;
+  height: $footer-height;
+  justify-content: center;
+  margin-top: 50px;
+}
+
+.footer-title {
+  color: $white;
+  font-size: $footer-text-big;
+  letter-spacing: 25px;
+  order: 1;
```

Comment:
You dont need to add this, just order the elements in the index.html file

Reply:
I think this is a good implementation since the idea behind css is to attack the layout without needing to do that through ordering tags in the html file.